react/features/shared-video/index.js

```diff
@@ -1,2 +1,5 @@
 export * from './actions';
 export * from './actionTypes';
+export * from './components';
```

Comment:
let's drop index.js while we're here, instead of adding more to it...

Reply:
File contains only these 3 exports right now.

Reply:
kill the entire file and fix the imports please :-)

Reply:
Killed